app/views/layouts/application.html.haml

```diff
@@ -8,6 +8,7 @@
     = javascript_pack_tag 'application'
     = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %link{:href => "https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css", :rel => "stylesheet"}/
```

Comment:
This `:href => ` is old syntax, update it.

Reply:
fixed